charts/opensearch/templates/securityconfig.yaml

```
@@ -0,0 +1,16 @@
+{{- if and .Values.securityConfig.config.securityConfigSecret .Values.securityConfig.config.data }}
+apiVersion: v1
+kind: Secret
+metadata:
+  name: {{ .Values.securityConfig.config.securityConfigSecret }}
+  namespace: {{ .Release.Namespace }}
+  labels:
+    app: { { .Chart.Name } }
+    release: { { .Release.Name } }
+    heritage: { { .Release.Service } }
```

This wasn't introduced in this PR, but it's a minor fix that I didn't want to get overlooked in the migration. The extra space between braces causes the chart to fail to parse if you're defining inline configuration in a single secret. Removing the extra space installs appropriately:
```suggestion
    app: {{ .Chart.Name }}
    release: {{ .Release.Name }}
    heritage: {{ .Release.Service }}
```

---

@bstrdsmkr Thanks for pointing this out. I will make the change.